src/main/java/mate/academy/rickandmorty/service/RickAndMortyClient.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.rickandmorty.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.external.CharacterResponseDataDto;
+import mate.academy.rickandmorty.dto.external.CharacterResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class RickAndMortyClient {
+    public static final String BASE_URL = "https://rickandmortyapi.com/api/character";
+    private final ObjectMapper objectMapper;
+
+    public List<CharacterResponseDto> getAllCharacters() {
```

Comment:
Add PostConstruct here and call save in the end of the method
```suggestion
   @PostConstruct
    public void initializeCharacters() {
```